pkg/util/encoding/encoding.go

```diff
@@ -2038,7 +2148,7 @@ func UpperBoundValueEncodingSize(colID uint32, typ Type, size int) (int, bool) {
 			return len(encodedTag) + maxBinaryUvarintSize + upperBoundNonsortingDecimalUnscaledSize(size), true
 		}
 		return 0, false
-	case Time:
+	case Time, TimeTZ:
 		return len(encodedTag) + 2*maxVarintSize, true
```

Comment:
This might be incorrect, I'm not sure. Because the tag is 17 (larger than 15) maybe the length is larger?